src/ms3/bs4_parser.py

```diff
@@ -1836,6 +1842,26 @@ def make_excerpt(
                 If the excerpt has chord labels, make sure the first label starts with the given local key, e.g.
                 'I' for the major tonic key or '#iv' for the raised subdominant minor key or 'bVII' for the lowered
                 subtonic major key.
+            start_mc_onset:
+                Onset value (either Fraction or float) specified as the "true" start of the first measure. Every note
+                with strictly smaller onset value will be "removed" (i.e. mutated into rest)
+            end_mc_onset:
+                Onset value (either Fraction or float) specified as the "true" end of the last measure. Every note
+                with strictly greater onset value will be "removed" (i.e. mutated into rest)
+            exclude_end:
+                If set to True, the note last note corresponding to ``end_mc_onset`` will also be "removed"
+            user_tempo: Optional[float], optional
+                The value that the user wants to set as the tempo of the excerpts. The tag will be added
+                to XML tree of the excerpt's file and will have the desired tempo
+            user_beat_unit: Optional[Fraction | float], optional
```

Comment:
This is unintuitive because we think of note values, and therefore beat units, as fractions of a whole note. Please accept the following values:
* 'quarters': `1/4`, `0.25`
* 'dotted quarters': `3/8`, `0.375`
* 'quarter triplet`: `1/6`, `0.166666`.
You will want to convert the input value using `Fraction(metronome_beat_unit).limit_denominator(32)` and multiply it by four (or equivalent).

Reply:
Hey @johentsch, do you think it would be a good idea to accept an optional string parameter as well that allows the user to define the beat unit with the corresponding label, i.e. `"quarters"`, `"dotted quarter"`, etc...?
(Is that what you already meant?)